backend/penndata/management/commands/get_venture_events.py

```diff
@@ -0,0 +1,112 @@
+import datetime
+import html
+
+import requests
+from bs4 import BeautifulSoup
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from penndata.models import Event
+
+
+VENTURE_EVENTS_WEBSITE = "https://venturelab.upenn.edu/venture-lab-events"
+HEADERS = {"User-Agent": "Mozilla/5.0 AppleWebKit/537.36 Chrome/91.0.4472.124 Safari/537.36"}
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        now = timezone.localtime()
+        current_month, current_year = now.month, now.year
+
+        try:
+            resp = requests.get(VENTURE_EVENTS_WEBSITE, headers=HEADERS)
+        except ConnectionError:
+            print("Error:", ConnectionError)
+            return None
+
+        soup = BeautifulSoup(resp.text, "html.parser")
+
+        event_containers = soup.find_all("div", class_="PromoSearchResultEvent")
+        last_start_datetime = None
+
+        for event in event_containers:
+            event_date_elem = event.find("div", class_="PromoSearchResultEvent-eventDate")
+            event_start_datetime = None
+            event_end_datetime = None
+            # some events don't have a start/end date time or a year
+            if event_date_elem:
+                event_date_str = event_date_elem.text.strip()
+
+                event_date_parts = event_date_str.split(" at ")
+                event_start_str = event_date_parts[1].split(" - ")[0].strip()
+                event_end_str = event_date_parts[1].split(" - ")[1].strip()
+
+                event_start_datetime = datetime.strptime(
+                    event_date_parts[0] + " " + event_start_str, "%B %d, %Y %I:%M%p"
+                )
+                event_end_datetime = datetime.strptime(
+                    event_date_parts[0] + " " + event_end_str, "%B %d, %Y %I:%M%p"
+                )
+                last_start_datetime = event_start_datetime
+            else:  # if no year given
+                event_month_elem = event.find("div", class_="PromoSearchResultEvent-month")
+                event_day_elem = event.find("div", class_="PromoSearchResultEvent-day")
+
+                if event_month_elem and event_day_elem:
+                    event_month = event_month_elem.text.strip()
+                    event_day = int(event_day_elem.text.strip())
+
```

Comment:
Makes a lot of sense, great work for solving out the logic + comments + making the code very readable!